sentence_transformers/sparse_encoder/SparseEncoder.py

```diff
@@ -1022,6 +999,34 @@ def max_seq_length(self, value) -> None:
         """
         self._first_module().max_seq_length = value
 
+    @property
+    def chunk_size(self) -> int | None:
+        """
+        Returns the chunk size of the SpladePooling module, if present.
+
+        Returns:
+            Optional[int]: The chunk size, or None if SpladePooling is not found or chunk_size is not set.
+        """
+        for mod in self._modules.values():
+            if isinstance(mod, SpladePooling):
+                return mod.chunk_size
+        logger.warning("SpladePooling module not found. Cannot get chunk_size.")
+        return None
+
+    @chunk_size.setter
+    def chunk_size(self, value: int) -> None:
+        """
+        Sets the chunk size of the SpladePooling module, if present.
+        """
+        found_splade = False
+        for mod in self._modules.values():
+            if isinstance(mod, SpladePooling):
+                mod.chunk_size = value
+                found_splade = True
+                break
+        if not found_splade:
```

Comment:
```suggestion
                break
        else:
```
The for-else is one of the least known about Python features, but it works great for exactly this. The `else` branch executes only if the `for` branch ended normally, i.e. without a `break`.

Reply:
Didn't know it at all, thanks !